config.json

```diff
@@ -4,6 +4,10 @@
   "blurb": "Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.",
   "gitter": "ruby",
   "solution_pattern": "[Ee]xample|\\.meta/solutions/[^/]*\\.rb",
+  "editor": {
```

Comment:
Perhaps this should be called `online_editor` or `recommended_editor_settings` or something. If it's just called `editor`, I expect to see at least a few bug reports per year surprised that this hasn't overridden someone's VS Code defaults.